_collections/_information/participation.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default-md
+title: Scout/Parent Participation
+subtitle: Scout and parent participation is crucial to the success of Troop 370. Troop 370 has weekly meetings, goes on monthly campouts, attends multiple summer camps, leads dozens of Eagle Scout service projects, and enjoys multiple high adventure opportunities throughout the year. We encourage every youth scout and adult scouter to participate in these wonderful opportunities provided by the Troop.
+nav_order: 3
+redirect_from: participation.html
+---
+## Scout Participation (Youth)
+
+Troop 370 is a boy-led troop, meaning that the scouts lead the meetings and plan the activities and campouts - not the adults. All scouts have the opportunity to take leadership positions in the troop, such as being a Senior Patrol Leader, Assistant Senior Patrol Leader, Instructor, Troop Guide, Patrol Leader, Scribe, Quartermaster, or Historian. Scouts can take leadership positions at any time or for any purpose; they do not need to wait until elections or for when they need it for advancement. One of the primary roles of leaders in the troop is planning the meetings in the monthly PLCs (patrol leaders' councils). Any scout wanting to give their input for the weekly meetings can do so by attending the PLCs. Read more about our leadership in our [Guidelines for Positions of Leadership document](/src/documents/Guidelines%20for%20Positions%20of%20Leadership.pdf).
+
+Advancement moves at the pace of the scout, so he can move as quickly or as slowly as he wants or as time permits. Troop instructors and assistant scoutmasters are happy to help any scout with the advancement requirements.
+
+Overall, there are many opportunities for fun, service, and leadership at the Troop, and scouts can participate in as many or as few activities and opportunities as they decide!
+
+## Scouter Participation (Adults)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/app/Troop_370/troop370/pullRequest?prid=4642321)